webpack.shared.config.js

```diff
@@ -26,21 +32,60 @@ module.exports = {
         // use: ExtractTextPlugin.extract(['css-loader', 'sass-loader']),
         use: ['css-hot-loader'].concat(ExtractTextPlugin.extract({
           fallback: 'style-loader',
-          use: ['css-loader', 'sass-loader'],
+          use: [
+            {
+              loader: 'css-loader',
+              options: { importLoaders: 1 }
+            },
+            {
+              loader: 'postcss-loader',
+              options: {
+                ident: 'postcss',
+                plugins: () => [ autoprefixer() ]
+              }
+            },
+            {
+              loader: 'sass-loader'
+            }
+          ]
         })),
+      },
+      {
+        test: /\.js$/,
+        enforce: 'pre',
```

Comment:
Noting the `enforce: 'pre'` here that makes this standalone in the rules (instead of being just another loader)